src/wdio/services/TerraCommands/chai-methods.js

```diff
@@ -20,6 +33,24 @@ function accessible() {
     `expected no accessibility violations but got:\n\t${errors[0]}`,
     'expected accessibility errors but received none',
   );
+
+  // The block of code below is used to display a warning when a test 
```

Comment:
@benbcai  check out the logic we use int eh visual regression configuration. We use are able to pull the test name and the test parent (describe block) to name the screenshots. I bet we can use some of that logic here: https://github.com/cerner/terra-toolkit/blob/master/config/wdio/visualRegressionConf.js#L31-L32

Reply:
This might be out of the scope here though...since this is a custom chai Assertion method (some [docs](https://www.chaijs.com/guide/helpers/#adding-methods)).

it might be we want to pass the test name down to the method and use it if/when we need to add a 'warning'